packages/desktop-electron/security.ts

```diff
@@ -1,15 +1,13 @@
-const electron = require('electron');
+import { app, session } from 'electron';
 
-electron.app.on('web-contents-created', function (event, contents) {
+app.on('web-contents-created', function (event, contents) {
   contents.on('will-attach-webview', function (event, webPreferences) {
-    delete webPreferences.preloadURL;
     delete webPreferences.preload;
 
     webPreferences.nodeIntegration = false;
     webPreferences.webSecurity = true;
     webPreferences.allowRunningInsecureContent = false;
     webPreferences.experimentalFeatures = false;
-    webPreferences.enableBlinkFeatures = false;
```

Comment:
This property takes a string array of feature flags, false isn't valid. Leaving it default means empty array which is gives it the same behavior.